index.html

```diff
@@ -32,15 +37,14 @@ <h2 class="subtitle">KENYA YOUTH SUMMIT <br>2022</h2>
       Kenya, come one ,come all, as we interact and share the youth agenda,It's out time and we want to network 
       and share our views.
       </p>
-    </div>
-    <div class="dates">
       <h3 class="day">2022/07/29~ sun/31</h3>
-      <h4 class="venue">@KICC ,Nairobi, Kenya </a></h4>
+      <h4 class="venue">@KICC ,Nairobi, Kenya </h4>
     </div>
   </header>
   <main>
     <section class="main_program">
       <h2 class="main_prog_title">Main Program</h2>
+      <div class="card_wrap_container">
       <div class="card_container">
         <div class="card_img">
           <img src="./assets/conference_icon.png" alt="conference"> 
```

Comment:
- [ ] It would be great if you could add the white border effect when a mouse hovers on each of the program cards 👍
- Required design:
![Annotation 2022-06-24 013044](https://user-images.githubusercontent.com/85113567/175440095-fc20b147-4572-4910-810d-b7ea69f45cc7.jpg)